dkg/dkg.go

```diff
@@ -478,8 +478,9 @@ func startSyncProtocol(ctx context.Context, tcpNode host.Host, key *k1.PrivateKe
 			break
 		}
 
-		// Sleep for 100ms to let clients connect with each other.
-		time.Sleep(time.Millisecond * 100)
+		// Sleep for 250ms to let clients connect with each other.
+		// Must be at least two times greater than the sync messages period specified in client.go NewClient().
```

Comment:
can you please point me to the code where this is specified?

Reply:
https://github.com/ObolNetwork/charon/blob/d2748d5c96dde7c790ae6a18b6e1ef8ecb8e37d7/dkg/sync/client.go#L41